test/pipelines/test_eval.py

```diff
@@ -75,11 +75,11 @@ def test_summarizer_calculate_metrics(document_store_with_docs: ElasticsearchDoc
     assert metrics["Retriever"]["precision"] == 1.0
     assert metrics["Retriever"]["ndcg"] == pytest.approx(0.9461, 1e-4)
     assert metrics["Summarizer"]["mrr"] == 1.0
-    assert metrics["Summarizer"]["map"] == 0.735
-    assert metrics["Summarizer"]["recall_multi_hit"] == 0.8
+    assert metrics["Summarizer"]["map"] == pytest.approx(0.9167, 1e-4)
+    assert metrics["Summarizer"]["recall_multi_hit"] == pytest.approx(0.9167, 1e-4)
     assert metrics["Summarizer"]["recall_single_hit"] == 1.0
-    assert metrics["Summarizer"]["precision"] == 0.8
-    assert metrics["Summarizer"]["ndcg"] == pytest.approx(0.8422, 1e-4)
```

Comment:
Since the document content no longer changes, the metrics for the Summarizer are the same as the Retriever.